### PR TITLE
Stylabling <Input />

### PR DESCRIPTION
**Pull request by danielstro**
Changed <Input /> to use stylable

### CODE REVIEW

File: packages/wix-ui-core/src/components/Input/Input.spec.tsx [hunk 1]
```diff
@@ -209,13 +209,13 @@ describe('Input', () => {
     });
   });
 
-  describe('testkit', () => {
+  describe.skip('testkit', () => {
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/Input.spec.tsx]: no need to skip. @amiryonatan fixed `isTestkitExists` and  `isEnzymeTestkitExists` functions

File: packages/wix-ui-core/src/components/Input/Input.st.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.root {
+    -st-states: disabled;
+    font-style: normal;
+    font-weight: normal;
+    text-decoration: none;
+    
+    box-sizing: border-box;
+    -webkit-font-smoothing: antialiased;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/Input.st.css]: I am pretty sure that  in stylable all of these vendor prefixers are being taken care of, so you can remove this line

File: packages/wix-ui-core/src/components/Input/InputStyle.st.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  borderColorDisabled: black;
+  borderColorHover: black;
+  borderColorFocus: black;
+  hoverBorderColor: black;
+  borderRadius: 6px;
+}
+
+.root {
+  -st-extends: Input;
+  font-family: "HelveticaNeueW01-45Ligh", "HelveticaNeueW02-45Ligh", "HelveticaNeueW10-45Ligh", "Helvetica Neue", "Helvetica", "Arial", "メイリオ, meiryo", "ヒラギノ角ゴ pro w3", "hiragino kaku gothic pro", "sans-serif";
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/InputStyle.st.css]: maybe this should sit in the resets?

File: packages/wix-ui-core/src/components/Input/InputStyle.st.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  font-size: 16px;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/InputStyle.st.css]: this is something that we would like to expose to be changed by mixin as well probably

File: packages/wix-ui-core/src/components/Input/InputStyle.st.css [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+.root:disabled:hover {
+  border-color: value(hoverBorderColor);
+}
+
+/* Placeholder color */
+.root::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+  color: value(colorPlaceholder);
+}
+.root:-moz-placeholder { /* Firefox 18- */
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/InputStyle.st.css]: don't think you need those ones with stylable. ask Barak or Ido

File: packages/wix-ui-core/src/components/Input/index.tsx [hunk 6]
```diff
@@ -158,10 +150,9 @@ class Input extends React.Component<InputProps, InputState> {
         tabIndex={tabIndex}
         type={type}
         value={value}
+        onMouseOver={console.log}
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Input/index.tsx]: onMouseOver?
and you forgot a console.log here

Comment (danielstro) [reply packages/wix-ui-core/src/components/Input/index.tsx]: Oh yeah, had a hover problem

File: packages/wix-ui-core/src/components/Input/InputStyle.st.css (hunk 3 above)
Comment (danielstro) [reply packages/wix-ui-core/src/components/Input/InputStyle.st.css]: Done

File: packages/wix-ui-core/src/components/Input/InputStyle.st.css (hunk 5 above)
Comment (danielstro) [reply packages/wix-ui-core/src/components/Input/InputStyle.st.css]: We still need it for now

File: packages/wix-ui-core/stories/index.tsx [hunk 7]
```diff
@@ -19,7 +20,6 @@ const dropdownOptions =
 
 require('./Badge').story();
 require('./Button').story();
-require('./Input').story();
```
Comment (shlomitc) [inline packages/wix-ui-core/stories/index.tsx]: should stay the same, just make sure that it's stylable compatible

**Pull request merged**